src/Forcings/Forcings.jl

```diff
@@ -32,6 +32,11 @@ function AtmosphereModels.materialize_atmosphere_model_forcing(forcings::Tuple,
 end
 
 # Handle compute_forcing! for MultipleForcings
-AtmosphereModels.compute_forcing!(mf::MultipleForcings) = compute_forcing!(mf.forcings)
+function AtmosphereModels.compute_forcing!(mf::MultipleForcings)
+    for forcing in mf.forcings
+        compute_forcing!(forcing)
+    end
+    return nothing
+end
```

Comment:
How did it work before? 🤔 Or it didn't?  Are we missing tests?

Reply:
i don't think it did.

Reply:
But then we're missing relevant tests?

Reply:
yes for sure

there is a test but it is too basic / loose, so it passed without this being caught.